f600233d1b154d507b782f84604bcb12/update/sys_script_include_9027655d97a7d5106fa8b4b3f153afba.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.AtatHelper</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>ATAT class that contains commonly used utility functions that can be used throughout the application.</description>
+        <name>AtatHelper</name>
+        <script><![CDATA[var AtatHelper = Class.create();
+AtatHelper.prototype = {
```

Comment:
Won't hold this up but generally we'll want these types of classes to have more specific names

Reply:
@jeffsegal-ccpo I'm open to name suggestions, just went with something.